hardware/diver/main/Core/Inc/banks/BankLayout.h

```diff
@@ -90,4 +93,38 @@ DiverBankBase* banks[] = {
             bool on = l.i % wavelength < pulsewidth;
             return (on != state.invert) * DAC_MAX_VALUE;
         }
+    ),
+    // Bank 8 - Dynamic Perlin Noise
+    new WavePlusLUT(
+        {.altA_default = 0.1f,
+         .altB_default = 0.1f,
+         .scrollrange_default = ScrollRange::Tortoise,
+         .deinterlace_mode = 0,
+         .update_style = WavePlusLUT::UpdateStyle::PerFrame},
+        [](WavePlusLUT::Lookup& l, DiverUIState& state)
+        {
+            float pos = float(l.i) / float(l.vres);
+            float freq = (64 * state.param_altA + 0.1) / MAX_SLIDER_VALUE;
+            int depth = int(12 * state.param_altB / MAX_SLIDER_VALUE + 1);
+            float n = perlin1d(pos, freq, depth, 4.0);
+            float val = state.invert ? std::abs(1 - n) : n;
+            return val * DAC_MAX_VALUE;
+        }
+    ),
+    // bank 9 - Sine wave
+    new WavePlusLUT(
+        {.altA_default = 0.1f,
+         .altB_default = 0.1f,
+         .scrollrange_default = ScrollRange::Hare,
+         .deinterlace_mode = 0,
+         .update_style = WavePlusLUT::UpdateStyle::PerFrame},
+        [](WavePlusLUT::Lookup& l, DiverUIState& state)
+        {
+            float pos = float(l.i) / float(l.vres);
+            float freq = l.vres * state.param_altA / MAX_SLIDER_VALUE;
+            float offset = 32 * state.param_altB / MAX_SLIDER_VALUE;
+            float sineWave = sin(pos * freq + offset);
```

Comment:
We might want to look at adding a sine function lookup table -- see e.g. how MI Rings approaches it:
https://github.com/pichenettes/eurorack/blob/84f4f67aaa25bf696093b224e2a51a5c18143e4f/rings/resources.cc
which is generated by the python script:
https://github.com/pichenettes/eurorack/blob/84f4f67aaa25bf696093b224e2a51a5c18143e4f/rings/resources/lookup_tables.py
and then used via an interpolation:
https://github.com/pichenettes/eurorack/blob/84f4f67aaa25bf696093b224e2a51a5c18143e4f/rings/dsp/fx/chorus.h#L70

We can probably get away with many many fewer samples. The choice of 8-bit or 16-bit is tough; 8-bit will have to be shifted to the 10-bit DAC output, but the table will consume much less memory. 16-bit would give a smoother approximation, but most of those extra bits would be wasted.

I believe that if we have a lookup table for `sin()`, you could also compute `cos()` by looking up in the same table with a phase offset.